Frameworks/UIKit/UIWebView.mm

```diff
@@ -251,6 +251,25 @@ - (NSString*)stringByEvaluatingJavaScriptFromString:(NSString*)string {
 }
 
 /**
+ @Status Caveat
+  @Notes This is a workaround. Original UIWebView does not have this method
+*/
+- (void)evaluateJavaScript:(NSString*)javaScriptString completionHandler:(void (^)(id, NSError*))completionHandler {
```

Comment:
ah, yes.  Inexplicably, I thought this function was "waiting" for the eval.  It isn't. Apologies for the false flag.

---
In reply to: [95208519](https://github.com/Microsoft/WinObjC/pull/1638#discussion_r95208519) [](ancestors = 95208519,95190774)